### PR TITLE
robustness: add 2 more log lines when persistClientReports

### PR DESCRIPTION
**Pull request by siyuanfoundation**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

**Review by fuweid — APPROVED** [review]

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @siyuanfoundation

File: tests/robustness/report/client.go [hunk 1]
```diff
@@ -58,9 +58,13 @@ func persistClientReports(t *testing.T, lg *zap.Logger, path string, reports []C
 		}
 		if len(r.Watch) != 0 {
 			persistWatchOperations(t, lg, filepath.Join(clientDir, "watch.json"), r.Watch)
+		} else {
+			lg.Info("skip persisting watch operations", zap.Int("client-id", r.ClientID))
```
Comment (serathius) [inline tests/robustness/report/client.go:62]:
```suggestion
			lg.Info("no watch operations for client, skip persisting", zap.Int("client-id", r.ClientID))
```

File: tests/robustness/report/client.go [hunk 2]
```diff
@@ -58,9 +58,13 @@ func persistClientReports(t *testing.T, lg *zap.Logger, path string, reports []C
...
 		}
 		if len(r.KeyValue) != 0 {
 			persistKeyValueOperations(t, lg, filepath.Join(clientDir, "operations.json"), r.KeyValue)
+		} else {
+			lg.Info("skip persisting KV operations", zap.Int("client-id", r.ClientID))
```
Comment (serathius) [inline tests/robustness/report/client.go:67]:
```suggestion
			lg.Info("no KV operations for client, skip persisting", zap.Int("client-id", r.ClientID))
```

**Review by serathius — APPROVED** [review]

**Pull request merged**